Set default of MVTLayer binary to true (#5800)
visgl/deck.gl: docs/whats-new.md

This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.

## deck.gl v8.5 (In development)

### MVT Layer

#### Direct binary parsing

> Mapbox Vector Tiles parsing throughput is now 2-3x faster, due to MVT tiles being parsed directly into binary attributes rather than GeoJSON, and additional work (including [triangulation](https://github.com/visgl/loaders.gl/blob/master/docs/whats-new.md#v30-in-development))being performed on worker threads. Speed comparison on some example data sets (MVT tiles parsed per second):

Reviewer: ```suggestion
Mapbox Vector Tiles parsing throughput is now 2-3x faster, due to MVT tiles being parsed directly into binary attributes rather than GeoJSON, and additional work (including [triangulation](https://github.com/visgl/loaders.gl/blob/master/docs/whats-new.md#v30-in-development)) being performed on worker threads. Speed comparison on some example data sets (MVT tiles parsed per second):
```

|  Data set                 | `binary: false` | `binary: true` | Speed increase |
| ------------------ | ----------- | ------ | -------------- |

> | Block groups       | 2.86/s      | 5.57/s | 1.94x          |
> | Census layer       | 6.09/s      | 11.9/s | 1.95x          |

Reviewer: ```suggestion
| Block Groups       | 2.86/s      | 5.57/s | 1.94x          |
| Census Layer       | 6.09/s      | 11.9/s | 1.95x          |
```
for consistent capitalization across lines

Author: Thanks for the corrections, Kyle :)

| Counties Layer     | 72.5/s      | 141/s  | 1.94x          |

> | Usa Zip Code Layer | 8.45/s      | 20.3/s | 2.4x           |

Reviewer: ```suggestion
| USA Zip Code Layer | 8.45/s      | 20.3/s | 2.4x           |
```

This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.

## deck.gl v8.5 (In development)

### MVT Layer

> #### Binary mode

Reviewer 2: Move this to upgrade-guide.md

Author: I take it you just mean this one section (`Binary mode`), and not the other two (`Direct binary parsing` and `Polygon triangulation in worker`)

Author: I've moved this now

> The `binary` prop of `MVTLayer` is set to `true` by default.

Reviewer 3: Yes, this line should go into upgrade guide.

#### Direct binary parsing

> Binary output is now 2-3X faster for large datasets thanks to parsing directly from PBF to binary, rather than going through GeoJSON as an intermediate representation. Speed comparison on some example data sets (MVT tiles parsed per second):

Reviewer 3: ```suggestion
Mapbox Vector Tiles parsing throughput is now 2-3x faster, due to MVT tiles being parsed directly into binary attributes rather than GeoJSON, and additional work being performed on worker threads. Speed comparison on some example data sets (MVT tiles parsed per second):
```

> |                    | Via GeoJSON | Direct | Speed increase |
> | ------------------ | ----------- | ------ | -------------- |
> | Block groups       | 2.86/s      | 5.57/s | 1.94X          |
> | Census layer       | 6.09/s      | 11.9/s | 1.95X          |
> | Counties Layer     | 72.5/s      | 141/s  | 1.94X          |
> | Usa Zip Code Layer | 8.45/s      | 20.3/s | 2.4X           |

Reviewer 3: ```suggestion
|  Data set                 | `binary: false` | `binary: true` | Speed increase |
| ------------------ | ----------- | ------ | -------------- |
| Block groups       | 2.86/s      | 5.57/s | 1.94x          |
| Census layer       | 6.09/s      | 11.9/s | 1.95x          |
| Counties Layer     | 72.5/s      | 141/s  | 1.94x          |
| Usa Zip Code Layer | 8.45/s      | 20.3/s | 2.4x           |
```

_Benchmarks ran using scripts on a 2012 MacBook Pro, 2.3 GHz Intel Core i7, 8 GB, measuring parsing time of MVTLoader only (network time and rendering is not included)_

> #### Polygon triangulation in worker

Reviewer 3: This seems like a loaders.gl release note, not sure it is needed here. We could just keep this in the previous section and include a link to the loaders.gl MVTLoader.

Author: Done
